### DISCUSSION TITLE
model benchmarks for various sampling strategies?

### DISCUSSION THREAD

**Discussion by okuvshynov** [ggml-org/llama.cpp#8038] [Q&A]
Let's say you pick a model (llama3-8b) and run evaluation of benchmarks typically reported by model creators (MMLU, HumanEval, etc.) with different sampling strategies (for example, do beam search with widths w1, w2, w3, ..) - how different would evaluation look like? 

Are there any existing results?
Thank you.

**Comment by ggerganov** [comment]
AFAIK benchmarks do not use sampling. Not sure what is HumanEval, but MMLU does not involve sampling

**Reply by okuvshynov** [reply]
Thank you! Sorry i missed your reply
